.github/workflows/integration.yml

```diff
@@ -29,8 +29,9 @@ jobs:
       fail-fast: false
       matrix:
        os: ["macos-latest", "ubuntu-latest"]
+       python-version: ["3.8", "3.11"]
```

Comment:
This seems specific so there's likely a reason, but out of curiosity, why do we only do 3.8 and 3.11? Is it just dbt Cloud and current? Or is there another reason?

Reply:
Do we feel fine with obscure 3.9 specific things coming up in regression reports as opposed to us catching it first? (serious ask)